tests/ui/test_ui_tools.py

```diff
@@ -2221,11 +2224,11 @@ def test_msg_generates_search_and_header_bar(
         "expected_header, to_vary_in_last_message",
         [
             (
-                [STATUS_INACTIVE, "alice", " ", "DAYDATETIME"],
+                [STATUS_INACTIVE, "alice(4)", " ", "DAYDATETIME"],
```

Comment:
I think we'll probably want a space between the name and user id.

Reply:
Okay, making that change in both test cases, and boxes.py